x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -533,7 +535,10 @@ m:d  | l:i
 ; 
 
 IfDuplicateNamesGroupingHasPriority
```

Comment:
This test is failing in BWC 9.0.0, expecting instead: `Line 1:37: Field 'languages' shadowed by field at line 1:89`. Verified it fails in local too.

I'm trying to find the cause, as column 89 is in the middle of "sort", and it doesn't make sense to me. Also, with the new capability, it shouldn't run at all (?)

Reply:
Oh well, in multi-cluster it gets transformed to: `FROM *:employees,employees  | stats languages = avg(height), languages = min(height) by languages | sort languages`

The other numbers weren't changed tho, so I may have done something wrong in the ctx change

Reply:
Ok, MultiClusterSpecIT modifies the warnings lines. But not those. So I'll see what can I do about it